# HG changeset patch
# Parent 1c9cd44cff88310a3303844476a9adea88c52fd8
win32: when using the msys-perl-wrapper (on $host==WINNT), always use the mozillabuild msys perl

diff --git a/build/msys-perl-wrapper b/build/msys-perl-wrapper
--- a/build/msys-perl-wrapper
+++ b/build/msys-perl-wrapper
@@ -13,4 +13,4 @@ do
     args="${args} '${i}'"
 done
 
-eval "exec perl $args"
+eval "exec ${MOZILLABUILD}/msys/bin/perl $args"
diff --git a/js/src/build/msys-perl-wrapper b/js/src/build/msys-perl-wrapper
--- a/js/src/build/msys-perl-wrapper
+++ b/js/src/build/msys-perl-wrapper
@@ -13,4 +13,4 @@ do
     args="${args} '${i}'"
 done
 
-eval "exec perl $args"
+eval "exec ${MOZILLABUILD}/msys/bin/perl $args"
